k8s/aat/common/reform-scan/blob-router.yaml

```diff
@@ -24,6 +24,9 @@ spec:
       environment:
         STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 0
         RESTART_ME: "To recreate pods"
+        SEND_DAILY_REPORT_ENABLED: "true"
+        SMTP_HOST: smtp.office365.com
+        SEND_DAILY_REPORT_CRON: "* */5 * ? * MON-FRI"
```

Comment:
for testing we can keep it an hour otherwise you will be bombarded with loads of email

Reply:
@nitinprabhu  Done